pkg/apigee/envVar.go

```diff
@@ -0,0 +1,115 @@
+package apigee
```

Comment:
That's a good point. I was somewhat unsure of where the best place for that stuff to live. I decided on the apigee package as it was using apigee specific structs. It could also live in enrober's `helper` package but I'm not sure I even like the idea of having a general `helper` package at all.